src/main/java/com/inventorysystem/inventory/VendorProfitController.java

```diff
@@ -23,7 +23,7 @@
     @RequestMapping("/")
     public String start(Model model) {
        Product product = new Product();
-       product.setProductID(1);
+       product.setProductSKU(1);
```

Comment:
I understand your rationale around SKU (Stock Keeping Unit), but that might not be the same thing as product ID in all cases.
Product ID: a unique identifier for a product.  This could also be a database-generated primary key, where SKU is a business key.
UPC (Universal Product Code: 12 digit), EAN (European Article Number: 13 digit), and PLU (Product Lookup: 4 digit) are all variations on SKU.
So yes, SKU is often the identifier for a product, but it may not be the only identifier... so product ID may or may not be a SKU.